ui.content/src/main/content/jcr_root/content/hclecomm/language-masters/en/.content.xml

```diff
@@ -2,6 +2,8 @@
 <jcr:root xmlns:sling="http://sling.apache.org/jcr/sling/1.0" xmlns:cq="http://www.day.com/jcr/cq/1.0" xmlns:jcr="http://www.jcp.org/jcr/1.0" xmlns:nt="http://www.jcp.org/jcr/nt/1.0"
     jcr:primaryType="cq:Page">
     <jcr:content
+        cq:cloudserviceconfigs="[/etc/cloudservices/generic-tracker/google-analytics]"
+        gaScript="https://www.googletagmanager.com/gtag/js?id=UA-173933021-1"
```

Comment:
@ShubhamGarg-AEM : same here

Reply:
@rishabh-behl  i have mentioned this in AG also, i can removed this. SHould i?